tests/models/test_responses.py

```diff
@@ -572,10 +572,7 @@ def test_iter_lines():
         200,
         content=b"Hello,\nworld!",
     )
-
-    content = []
-    for line in response.iter_lines():
-        content.append(line)
+    content = [line for line in response.iter_lines()]
```

Comment:
We could also use `content = list(response.iter_lines())` here.

Reply:
https://pypi.org/project/flake8-comprehensions/